packages/story-editor/src/app/history/historyProvider.tsx

```diff
@@ -17,14 +17,14 @@
 /**
  * External dependencies
  */
-import PropTypes from 'prop-types';
 import {
   useCallback,
   useEffect,
   useState,
   useRef,
 } from '@googleforcreators/react';
 import { useGlobalKeyDownEffect } from '@googleforcreators/design-system';
```

Comment:
@barklund / @swissspidy Any thoughts on why this error happens here:
`Cannot find module '@googleforcreators/design-system' or its corresponding type declarations.`

Not sure what I'm missing here 🤔 It was ok just before.

Reply:
Looks like needed to include `index.ts` as well to the `tsconfig.json`, however, then all the untyped errors come out. Had to remove the `design-system` as a reference [2017585](https://github.com/GoogleForCreators/web-stories-wp/pull/12674/commits/201758538ff969e90b962ba6aa9d3b0abe0e4356), however, then the module is not found again 🤷

Reply:
Ok, reverted that last change.

Basically now the `keyboard` part is typed, `design-system` is in the general `tsconfig.json` and also in the `story-editor` package's config, however, the error is still there since the `index.ts` file is not included in the `design-system`'s `tsconfig` file (if we include it, all the errors show up).

Perhaps now we can use `ts-ignore` for the module not found issue in this file (until the `design-system` package gets typed since the relevant part is actually typed? Thoughts, @swissspidy?

Reply:
Having the same problem [here](https://github.com/GoogleForCreators/web-stories-wp/actions/runs/3481863339/jobs/5823491322#step:10:9) as well.

Reply:
@miina The issue is that the `types` field in the `design-system` `package.json` points to `dist-types/index.d.ts` but no such file exists yet since `index.ts` is not listed in `tsconfig.json` (which is fine).

In 68bd6bf3b97c63a287c2d624b91bd9ab08921279 I added a quick workaround for that which should resolve this.

Thanks for converting all the keyboard stuff by the way 🚀